lib/components/formik/Switch.js

```diff
@@ -6,14 +6,11 @@ import Switch from "../Switch";
 export default function FormikSwitch({ name, ...rest }) {
   return (
     <Field name={name}>
-      {({ field, form: { errors }, meta }) => (
-        <Switch
-          name={name}
-          error={errors[field.name]}
-          checked={field.value}
-          {...rest}
-          {...field}
-        />
+      {({ field, meta: { error } }) => (
+        <>
+          <Switch name={name} checked={field.value} {...rest} {...field} />
+          {error && <div className="nui-input__error">{error}</div>}
```

Comment:
@piyushsi _a The error prop has to be passed to the Switch component and the error div should be there. This will ensure that a person who is not using formik also has support for error handling.

Check out the Select Component.